src/main/java/com/google/sps/data/DataManager.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+public interface DataManager {
+  public void storeUser(User user);
```

Comment:
For each method in the Interface class, we should also put the javadoc to explain how to use this method. You can copy and paste the javadoc from the implementation class to this interface class. The implementation class might contains some extra information (e.g. implementation details).